examples/batch_example.js

```diff
@@ -0,0 +1,58 @@
+const epsagon = require('../src/index');
+const http = require('http');
+
+
+epsagon.init({
+    token: process.env.EPSAGON_TOKEN,
+    appName: 'batch-test',
+    metadataOnly: false,
+    sendBatch: true,
+    batchSize: 1,
+    maxBatchSizeBytes: 5000000
```

Comment:
IMO better to use in KB (and to update the var name in this case)

Reply:
We already use Bytes in the regular sending method... that's for the sake of uniformity